__init__.py

```diff
@@ -23,12 +30,17 @@ def __init__(self, config):
         except KeyError as e:
             _LOGGER.error("Missing auth token! You must set 'github-token' in your config")
         self.name = self.config.get("name", "github")
+        # had troubles using the python 3.6 syntax, using asyncio as the package
+        loop = asyncio.get_event_loop()
+        raw_json = loop.run_until_complete(ConnectorGitHub._getSite('https://api.github.com/user?access_token='+self.github_token))
```

Comment:
This line is a little long. Please break it up a little. And don't forget your [PEP8 spacing](https://www.python.org/dev/peps/pep-0008/#whitespace-in-expressions-and-statements) :).